contracts/Billing.sol

```diff
@@ -37,11 +37,18 @@ contract Billing is IBilling, Governed, Rescuable {
      * @dev User adds tokens
      */
     event TokensAdded(address indexed user, uint256 amount);
+
     /**
      * @dev User removes tokens
      */
     event TokensRemoved(address indexed from, address indexed to, uint256 amount);
```

Comment:
I wonder if there is value in using different events for token removals on L1 and L2.

Reply:
mm scratch that, we can always filter that out at the subgraph level if we need to.